config/sync/upd_featured_case_studies.settings.yml

```diff
@@ -0,0 +1,5 @@
+featured_case_study1: '25'
+featured_case_study2: '13'
+featured_case_study3: '12'
```

Comment:
Only problem with setting these is if they change them will it not then get reverted on a configuration import as they are changing as content so somehow shouldn't really be saved in config?